docker/images/node/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN npm install --no-optional
 # Run it to check if `gulp build` works.
 # There was problems with it. See stb#258 github issue.
 RUN gulp build
+RUN echo 'bugaga' >> bu
 
 # Keeps track of changes if static files are built, otherwise builds them
-CMD bash -c "[ '$(ls -A /usr/app/src/build)' ] && gulp watch || (gulp build && gulp watch)"
+CMD bash -c "[ `ls -A /usr/app/src/front_build` ] && gulp watch || (gulp build && gulp watch)"
```

Comment:
Seems it will not work and it would be nice to verify this

Reply:
i'll revert to old code and create task about it. Old line `ls -A /usr/app/src/build` should be broken some times ago, because we changed `build` to `front_build`